packages/x-date-pickers/src/MobileDatePicker/tests/MobileDatePicker.test.tsx

```diff
@@ -124,12 +124,6 @@ describe('<MobileDatePicker />', () => {
     });
   });
 
-  it('prop `defaultCalendarMonth` – opens on provided month if date is `null`', () => {
```

Comment:
Nitpick: we could refactor the test to use `referenceDate` instead. 🤔

Reply:
We already have a similar test applied to all the pickers for `referenceDate` :+1:

Reply:
Cool. 👌